_sass/blocks/case-studies/_content.scss

```diff
@@ -46,10 +46,30 @@
   }
 }
 
-.case-studies-maps {
+.case_studies-maps {
   display: block;
 
   eiti-map {
     display: block;
   }
+
+  &.desktop {
+    display: none;
+  }
+
+  &.mobile {
+    display: block;
+    width: 50vh;
+  }
+
+  @include respond-to(medium-up) {
```

Comment:
Expected item on line 65 to appear before line 52. Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets